bb/master/master.cfg

```diff
@@ -156,8 +156,9 @@ c["change_source"] = []
 
 def pull_request_filter(pull_request_message):
     # Include only changes from fork repositories
-    if pull_request_message["head"]["repo"]["full_name"] == config.GITHUB_REPOSITORY:
-        return False
+    if pull_request_message["head"]["repo"] is not None:
+        if pull_request_message["head"]["repo"]["full_name"] == config.GITHUB_REPOSITORY:
```

Comment:
can be just:
if pull_request_message["head"]["repo"] and pull_request_message["head"]["repo"]["full_name"] == config.GITHUB_REPOSITORY: